WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailLikesView.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+class ReaderDetailLikesView: UIView, NibLoadable {
+
+    @IBOutlet weak var avatarStackView: UIStackView!
+    @IBOutlet weak var summaryLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyStyles()
+    }
+
+}
+
+private extension ReaderDetailLikesView {
+
+    func applyStyles() {
+        // Set border on all the avatar views
+        for subView in avatarStackView.subviews {
+            subView.layer.borderWidth = 1
+            subView.layer.borderColor = UIColor.white.cgColor
```

Comment:
Might want to use something like `UIColor.basicBackground.cgColor` so that this adapts for dark mode.

Reply:
Woops. Thank you!